### DISCUSSION TITLE
MacOS menu bar icon is stuck in top left of menu bar

### DISCUSSION THREAD

**Discussion by will09122000** [rustdesk/rustdesk#7084] [General]
### Bug Description

MacOS menu bar icon is stuck in top left of menu bar, on top of the apple icon. It can't be clicked on.

### How to Reproduce

Launch RustDesk on a Mac with a notch, with enough menu icons such that the icons get cut off from the notch.

### Expected Behavior

The icon should be in the menu bar on the left hand side.

### Operating system(s) on local side and remote side

MacOS 14.3 (23D56)

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

<img width="1512" alt="SCR-20240207-rtkn" src="https://github.com/rustdesk/rustdesk/assets/41238606/62172eca-73e9-4e14-a475-9640639ea1f5">

**Comment by lxne** [comment]
I do have the same issue on two different macbook pros.
Why did you put this here and not under issues?
Shouldn't it be moved there?

Oh, I just saw it originally was an issue, but was converted.
Why?

**Comment by xlionjuan** [comment]
Second this, happened on 1.2.4 nightly

**Comment by rustdesk** [comment]
What condition you can see this? I have attempted many time on my 14.5, failed to reproduce. https://github.com/rustdesk/rustdesk/discussions/8569

**Comment by rustdesk** [comment]
close to favor https://github.com/rustdesk/rustdesk/discussions/8569

So far it happens on intel mac 14.x, 12.6 does not have this issue. https://github.com/rustdesk/rustdesk/discussions/8569#discussioncomment-9936846
